handler.go

```diff
@@ -153,6 +169,57 @@ func (h HandlerOpts) GetSigningKeyFallback() string {
 	return *h.SigningKeyFallback
 }
 
+// GetAPIOrigin returns the host to use for sending API requests
+func (h HandlerOpts) GetAPIOrigin() string {
+	if h.isDev() {
+		return devServerOrigin
+	}
+
+	if h.APIOrigin == nil {
+		origin := os.Getenv("INNGEST_API_ORIGIN")
+		if origin != "" {
+			return origin
+		}
+
+		return defaultAPIOrigin
+	}
+
+	return *h.APIOrigin
+}
+
+// GetEventAPIOrigin returns the host to use for sending events
+func (h HandlerOpts) GetEventAPIOrigin() string {
+	if h.isDev() {
+		return devServerOrigin
```

Comment:
Users should still be able to override the Dev Server origin with the option or env var